src/map.rs

```diff
@@ -114,18 +114,18 @@ impl Map {
 }
 
 /// An iterator that iterates over all the layers in a map, obtained via [`Map::layers`].
-pub struct LayerIter<'map> {
+pub struct MapLayerIter<'map> {
```

Comment:
Changed name to disambiguate from new GroupLayerIter

Reply:
Hah, no need to apologize.
I must have picked the worst time to do that first PR yesterday!